### PR TITLE
gnome-icon-theme-symbolic: cleanup build environment and correct samp…

### PR DESCRIPTION
**Pull request by klausz65**
…le-manifest

### CODE REVIEW

File: components/desktop/gnome-icon-theme-symbolic/Makefile [hunk 1]
```diff
@@ -26,21 +27,14 @@ COMPONENT_ARCHIVE_HASH=	\
...
 
-include $(WS_MAKE_RULES)/prep.mk
-include $(WS_MAKE_RULES)/configure.mk
-include $(WS_MAKE_RULES)/ips.mk
+include $(WS_MAKE_RULES)/common.mk
 
 # Missing files in build dir for configure without this.
 COMPONENT_PRE_CONFIGURE_ACTION =        ($(CLONEY) $(SOURCE_DIR) $(@D))
 
 CONFIGURE_OPTIONS+=	--sysconfdir=/etc
```
Comment (mtelka) [inline components/desktop/gnome-icon-theme-symbolic/Makefile]: I assume this is not needed.

File: components/desktop/gnome-icon-theme-symbolic/manifests/sample-manifest.p5m [hunk 2]
```diff
@@ -10,10 +10,11 @@
 #
 
 #
-# Copyright 2016 <contributor>
+# Copyright 2024 <contributor>
 #
 
 set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.human-version value=$(HUMAN_VERSION)
```
Comment (mtelka) [inline components/desktop/gnome-icon-theme-symbolic/manifests/sample-manifest.p5m:17]: I assume the real manifest needs an update too.

File: components/desktop/gnome-icon-theme-symbolic/Makefile (hunk 1 above)
Comment (klausz65) [reply components/desktop/gnome-icon-theme-symbolic/Makefile]: done

File: components/desktop/gnome-icon-theme-symbolic/manifests/sample-manifest.p5m (hunk 2 above)
Comment (klausz65) [reply components/desktop/gnome-icon-theme-symbolic/manifests/sample-manifest.p5m:17]: done - and thanks for the heads-up

**Comment by klausz65** [comment]
will re-new it

**Pull request closed without merging**